packages/common/src/components/StandardPage/ResultStates.tsx

```diff
@@ -12,56 +11,60 @@ import {
 } from '@patternfly/react-core';
 import { ExclamationCircleIcon, SearchIcon } from '@patternfly/react-icons';
 
-export const ErrorState = () => {
-  const { t } = useTranslation();
+export const ErrorState = ({ title }: { title: string }) => {
   return (
     <EmptyState>
       <EmptyStateIcon icon={ExclamationCircleIcon} color="#C9190B" />
       <Title headingLevel="h4" size="lg">
-        {t('Unable to retrieve data')}
+        {title}
       </Title>
     </EmptyState>
   );
 };
 
-export const Loading = () => {
-  const { t } = useTranslation();
+export const Loading = ({ title }: { title: string }) => {
```

Comment:
looks like this can be refactored to a `<StandardEmptyState title={title} variant={container} icon={ {icon: null, color: null, variant: "container", component: Spinnner} } />`

can you make a refactor followup ?

Reply:
created #399